doc/developing-the-website.md

```diff
@@ -13,5 +13,14 @@ Alternatively, to dynamically rebuild the site and refresh the browser _when cha
     node_modules/.bin/gulp watch
 
 ### Next steps
+When you are done with developing, you can [build and deploy](building-and-deploying-the-website.md) using the automated steps.
 
-When you are done with developing you can [build](building-the-website.md) and [deploy](deploying-the-website.md).
\ No newline at end of file
+You can also [build](building-the-website.md) and [deploy](deploying-the-website.md) manually. In case Travis build breaks, you may want to build and deploy manually. 
+
+Here are a few use cases that Travis auto build and deploy doesn't cover:
```

Comment:
Are these "developnig the website" tasks?I suggest to move this over to `building-and-deploying-the-website.md` in a new subheadline "Not covered by automated build and deploy" (or similar). Here this is a